src/main/cpp/sequencer.cpp

```diff
@@ -153,9 +153,8 @@ namespace Sequencer
                 int eventStart = audioEvent->getEventStart();
                 int eventEnd   = audioEvent->getEventEnd();
 
-                if ( audioEvent->isLoopeable() ||
```

Comment:
Bug that has been slumbering for 3 years. This would imply that ANY sequenced event that was loopeable would ALWAYS be audible.